modules/platforms/dotnet/Apache.Ignite.Core/IgniteConfiguration.cs

```diff
@@ -1322,6 +1342,12 @@ public bool IsDaemon
         [SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
         public IDictionary<string, object> UserAttributes { get; set; }
 
+        /// <summary>
+        /// Gets or sets the affinity attributes for this node.
+        /// </summary>
+        [SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
+        public IDictionary<string, string> AffinityAttributes { get; set; }
```

Comment:
Thanks for adding the property on .NET side! Can you please also update `BaselineNode.cs` and add some checks to `PersistenceTest.TestBaselineTopology` to verify attribute propagation?